gitless/core.py

```diff
@@ -103,7 +103,7 @@ def __init__(self):
     self.git_repo = pygit2.Repository(path)
     self.remotes = RemoteCollection(self.git_repo.remotes, self)
     self.path = self.git_repo.path
-    self.root = self.path[:-6]  # strip trailing /.git/
+    self.root = self.path[:-6] if self.path[-5:-1] == '.git' else self.path # strip trailing /.git/
```

Comment:
Out of curiosity, does this mean that in Windows the `git_repo.path` variable that comes from pygit2 doesn't include a trailing `/.git/`?